### PR TITLE
feat(onboard/contributing install): create installation instructions …

### PR DESCRIPTION
**Pull request by gael-boyenval**
…for contributors

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/adeo/design-system--styleguide/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

create installation instruction and remove them from the readme

### CODE REVIEW

File: src/pages/Contributing/Install/index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+`git clone git@github.com:adeo/design-system--styleguide.git`
+
+move to the repo :  
+`cd design-system--styleguide`
+
+install dependencies and create symlink between packages :  
+`yarn lerna`
+
+Add a github personal access tokens :  
+[follow those instruction](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) and select `repo` as a scope for you rpersonnal access token.
```
Comment (tiloyi) [inline src/pages/Contributing/Install/index.md]: ~~rpersonnal~~ => personal

File: src/pages/Contributing/Install/index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+It also create symlink between the individual packages nodes_modules so they can be used in one another as npm dependencies.
+
+When releasing the design system, lerna will automaticaly generate changelogs for the global/parent repo, as well as for the individual packages and select a version number based on the commit syntax. It's why it is very important that you follow the [contributing guidelines](https://github.com/adeo/design-system--styleguide/blob/master/CONTRIBUTING.md).
+
+please [read the docs](https://github.com/lerna/lerna/) to learn more about it.
+
+### Clone the repo and start the development environement
+
+clone the repo :  
+`git clone git@github.com:adeo/design-system--styleguide.git`
```
Comment (tiagoux) [inline src/pages/Contributing/Install/index.md]: @gael-boyenval old address

File: src/pages/Contributing/Install/index.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+move to the repo :  
+`cd design-system--styleguide`
```
Comment (tiagoux) [inline src/pages/Contributing/Install/index.md]: old folder naming

File: src/pages/Contributing/Install/index.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+stylelint `fix` and `lint` are run at pre-commit, so you can't commit uggly stuffs ;).
+
+## Create a production build :
+
+`yarn build`
+
+## CI/CD
+
+If you want more explanation go to the related [wiki page](https://github.com/adeo/design-system--styleguide/wiki/CI-CD).
```
Comment (tiagoux) [inline src/pages/Contributing/Install/index.md]: old link

**Pull request merged**